protoc-gen-swagger/main.go

```diff
@@ -43,8 +43,12 @@ func main() {
 
 	glog.V(1).Info("Processing code generator request")
 	f := os.Stdin
-	if *file != "stdin" {
-		f, _ = os.Open("input.txt")
+	if *file != "-" {
+		var err error
+		f, err = os.Open(*file)
```

Comment:
@achew that would define a new f instead of assigning to the one created previously.